main.ts

```diff
@@ -1,13 +1,25 @@
 import { Command } from "cliffy";
 
+import { config } from "@/api/mod.ts";
+import { setupLogger } from "@/jcli/logger.ts";
+
+import { action as globalOptionAction } from "@/args.ts";
+
 import { default as projectsCommand } from "@/subcommands/projects/mod.ts";
 import { default as migrationsCommand } from "@/subcommands/migrations/mod.ts";
 import { default as functionsCommand } from "@/subcommands/functions/mod.ts";
+import { default as adminCommand } from "@/subcommands/admin/mod.ts";
```

Comment:
可以在 `@/subcommands/admin/mod.ts` 使用命名导出 这里就不用重命名了
```suggestion
import { adminCommand } from "@/subcommands/admin/mod.ts";
```
或者使用默认导入的方式也可以这样
```suggestion
import adminCommand from "@/subcommands/admin/mod.ts";
```

Reply:
好的，我改一下